README.md

```diff
@@ -10,14 +10,13 @@ displaying information from those in a centralized place.
 
 ## Table of Contents
 
-  - [Cloning and getting into the Project Dir](#cloning-and-getting-into-the-project-dir)
-  - [Setting up the development environment](#setting-up-the-development-environment)
-  - [Starting the dev server](#starting-the-dev-server)
-     - [Building](#building)
-     - [Deploying to gh-pages](#deploying-to-gh-pages)
-  - [Launching testsuite](#launching-testsuite)
-
-
+* [Cloning and getting into the Project Dir](#cloning-and-getting-into-the-project-dir)
+* [Setting up the development environment](#setting-up-the-development-environment)
+* [Starting the dev server](#starting-the-dev-server)
+  * [Building](#building)
+  * [Deploying to gh-pages](#deploying-to-gh-pages)
+* [Launching testsuite](#launching-testsuite)
+* [Linting](#linting)
```

Comment:
Seems you prettier'd this file too ;) I would be +1 on also auto formatting the .md and .json files as you suggested!

Reply:
Yeah, I didn't even notice. But linting rules for non-runtime code just hurts my soul. It's too anal.